Rocket.Chat/Resources/de.lproj/Localizable.strings

```diff
@@ -194,4 +194,11 @@
 "myaccount.settings.title" = "Einstellungen";
 "myaccount.settings.contactus" = "Kontaktiere uns";
 "myaccount.settings.license" = "Lizenz";
+"myaccount.settings.appicon" = "Application Icon"; // TODO:
 "myaccount.settings.version" = "Version: %@ (%@)";
+
+// Change Application Icon
+"myaccount.settings.changeicon.title" = "Choose Icon"; // TODO:
+"myaccount.settings.changeicon.header" = "You can change how your Rocket.Chat+ icon looks like on your device. Please choose one of icons below, whichever you like the most."; // TODO:
+"myaccount.settings.changeicon.errortitle" = "Cannot change icon"; // TODO:
+"myaccount.settings.changeicon.iosversion" = "Alternate application icons are not supported on iOS version below 10.3"; // TODO:
```

Comment:
"myaccount.settings.changeicon.title" = "Icon auswählen";
"myaccount.settings.changeicon.header" = "Du kannst eines der unterstehenden Icons für die App wählen, welches auf deinem Homescreen angezeigt wird.";
"myaccount.settings.changeicon.errortitle" = "Konnte App Icon nicht ändern";
"myaccount.settings.changeicon.iosversion" = "Alternative App Icons werden nicht von iOS älter als 10.3 unterstützt.";

Reply:
Thank you!